### PR TITLE
Add Key People Section on Home

### PR DESCRIPTION
**Pull request by VishalZ123**
Adds key people ( General Secretary Senate and General Secretary ACAC ) on the Home page under the Boards.
Used the OfficeBearerCard component.
key_people

### CODE REVIEW

File: acac_backend/src/main/models.py [hunk 1]
```diff
@@ -152,10 +152,11 @@ class Senate(models.Model):
     cover = VersatileImageField('Cover', upload_to='society_%Y', help_text="Upload high quality picture")
     skin = models.CharField(max_length=32, choices=SKIN_CHOICES, blank=True, default='mdb-skin',
                             help_text="Choose a skin while displaying senate page.")
-    members = models.ManyToManyField(UserProfile, through='SenateMembership', through_fields=('senate', 'userprofile'))
-    coordinator_student = models.ForeignKey(Faculty, blank=True, null=True, default=None,
-                                            on_delete=models.SET_NULL)
-    report_link = models.URLField(help_text='Add a drive link of annual report to show on board page', null=True, blank=True)
+    gen_secy_senate = models.ForeignKey(UserProfile, related_name='gen_secy_senate', limit_choices_to={'user__is_staff': True},
```
Comment (DarshitJain04) [inline acac_backend/src/main/models.py:155]: You forgot to remove `coordinator_student` and add `gen_secy_senate` and `gen_secy_acac` in `senatefixture.py` file

**Review by DarshitJain04 — CHANGES_REQUESTED** [review]

**Comment by VishalZ123** [comment]
@DarshitJain04 I changed the ```senatefixture.py``` file

**Review by DarshitJain04 — APPROVED** [review]

**Comment by DarshitJain04** [comment]
> Adds key people ( General Secretary Senate and General Secretary ACAC ) on the Home page under the Boards. Used the OfficeBearerCard component. key_people

It seems there is some spacing issue on the home page for these cards (probably for small screen size only). In such cases, it would be better to have 3 or 2 cards in a row?
image

**Comment by VishalZ123** [comment]
It was not the issue of small screen size, it was just rendering a new row of cards just after a card, I have corrected it, now it is similar to the Key People Section of ```Boards```






@DarshitJain04

**Pull request merged**